diff --git a/dist/esm/development/routing/config.js b/dist/esm/development/routing/config.js
index 30f2c0e03cf2c854a3de2ca52a2ff776825cf11c..6ce7a8038b186e104ed276fbadf82cd5072b6701 100644
--- a/dist/esm/development/routing/config.js
+++ b/dist/esm/development/routing/config.js
@@ -9,7 +9,7 @@ function receiveRoutingConfig(input) {
 }
 function receiveLocaleCookie(localeCookie) {
   return localeCookie ?? true ? {
-    name: 'NEXT_LOCALE',
+    name: 'LOCALE',
     sameSite: 'lax',
     ...(typeof localeCookie === 'object' && localeCookie)
 
diff --git a/dist/esm/production/routing/config.js b/dist/esm/production/routing/config.js
index ee4627826005b52911602a4ffab0b9ab202b5c63..dbfa5974a586c364ce9fd26657fac1d03b13a106 100644
--- a/dist/esm/production/routing/config.js
+++ b/dist/esm/production/routing/config.js
@@ -1 +1 @@
-function e(e){return{...e,localePrefix:(a=e.localePrefix,"object"==typeof a?a:{mode:a||"always"}),localeCookie:(o=e.localeCookie,!!(o??1)&&{name:"NEXT_LOCALE",sameSite:"lax",..."object"==typeof o&&o}),localeDetection:e.localeDetection??!0,alternateLinks:e.alternateLinks??!0};var o,a}export{e as receiveRoutingConfig};
+function e(e){return{...e,localePrefix:(a=e.localePrefix,"object"==typeof a?a:{mode:a||"always"}),localeCookie:(o=e.localeCookie,!!(o??1)&&{name:"LOCALE",sameSite:"lax",..."object"==typeof o&&o}),localeDetection:e.localeDetection??!0,alternateLinks:e.alternateLinks??!0};}export{e as receiveRoutingConfig};
